Lib/importlib/_bootstrap_external.py

```diff
@@ -906,8 +906,13 @@ class ExtensionFileLoader(FileLoader, _LoaderBasics):
 
     """
 
-    def __init__(self, name, path):
-        self.name = name
+    def __init__(self, fullname=None, path=None, *, name=None):
+        if name is not None:
+            _warnings.warn("the 'name' parameter is deprecated; use "
+                           "'fullname' instead", DeprecationWarning)
```

Comment:
You need to pass `stacklevel=2` in order to make `DeprecationWarning` more useful.